providers/typetalk/typetalk_test.go

```diff
@@ -0,0 +1,53 @@
+package typetalk_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/markbates/goth"
+	"github.com/markbates/goth/providers/typetalk"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_New(t *testing.T) {
+	t.Parallel()
+	a := assert.New(t)
+	p := provider()
+
+	a.Equal(p.ClientKey, os.Getenv("TYPETALK_KEY"))
+	a.Equal(p.Secret, os.Getenv("SLACK_SECRET"))
```

Comment:
This should be changed to `os.Getenv("TYPETALK_SECRET")`. I know it's doesn't matter in this test case, but it's helpful for the tests to look as "real" as possible in case someone comes looking through here for implementation details.